templates/api/keras_nlp/models/index.md

```diff
@@ -0,0 +1,65 @@
+# KerasNLP Models
+
+KerasNLP contains end-to-end implementations of popular model
+architectures. These models can be created in two ways:
+
+1) Through the `from_preset()` constructor, which instantiates an object with
+a pre-trained configurations, vocabularies, and (optionally) weights. Available
+preset IDs are listed on this page.
+
+2) Through custom configuration controlled by the user. To do this, simply
```

Comment:
Show a short code snippet to illustrate this for e.g. a classifier, a backbone and a preprocessor